Update docs for version 0.9.9 (#278)
AequilibraE/qaequilibrae: docs/source/menus_in_detail/readme.rst

    :alt: aeq menu

Most screenshots in this section are related to Sioux Falls, a standard example
in transportation network algorithm studies. 

> In the following sections, we present an image-based example on a realistic modelling workflow

Reviewer: ```suggestion
In the following sections, we present an illustrated example of a realistic modeling workflow
```
